Cronner/DI/CronnerExtension.php

```diff
@@ -105,6 +108,9 @@ public function beforeCompile()
 		foreach (array_keys($builder->findByTag(self::TASKS_TAG)) as $serviceName) {
 			$runner->addSetup('addTasks', array('@' . $serviceName));
 		}
+
+		$builder->getDefinition('tracy.bar')
+			->addSetup('addPanel', array($this->prefix('@bar')));
```

Comment:
Use detection of installed Nette\Tracy because it is not required to run Cronner.